### ISSUE TITLE
edgectl fail to login

### ISSUE THREAD

**Issue by katlimruiz** [datawire/ambassador#2754]
**Describe the bug**
$ edgectl login --namespace=ambassador myapi.com
panic: CmdUsage is not empty: map[upgrade:0xc000668c80]

goroutine 1 [running]:
github.com/datawire/ambassador/internal/pkg/edgectl.NewCmdUsage(0xc0006fd180, 0xc000695f00, 0x2, 0x2, 0xc000695f20)
        github.com/datawire/ambassador@/internal/pkg/edgectl/cmd_usage.go:46 +0x91c
main.main()
        github.com/datawire/ambassador@/cmd/edgectl/main.go:54 +0x143

**To Reproduce**
1. Go to Command Prompt
2. $ edgectl login --namespace=ambassador myapi.com
3. See error

**Expected behavior**
Login to api gateway

**Comment by katlimruiz** [comment]
how can I get ambassador version without entering to the console? kubectl describe?

**Comment by cakuros** [comment]
yes, if you `kubectl describe ambassador -n ambassador`, there should be a field that says `image`.  The version number after that is the version number.  What does `edgectl version` say?

**Comment by PaulStanos** [comment]
I'm having the same issue with edgectl on Windows. It seems like no matter what command I run I always get the `CmdUsage is not empty` error. I put edgectl.exe into 'C:/edgectl/' and added 'C:/edgectl/' to my PATH environment variable. I am running the commands from Git Bash, but the same error happens in Windows Command Prompt too. This is what I get running `edgectl version`:

```
edgectl version
panic: CmdUsage is not empty: map[upgrade:0xc000588500]

goroutine 1 [running]:
github.com/datawire/ambassador/internal/pkg/edgectl.NewCmdUsage(0xc0001f1b80, 0xc000737f00, 0x2, 0x2, 0xc000737f20)
        github.com/datawire/ambassador@/internal/pkg/edgectl/cmd_usage.go:46 +0x91c
main.main()
        github.com/datawire/ambassador@/cmd/edgectl/main.go:54 +0x143
```

Seems like the latest release of `edgectl` on Windows is broken?

**Comment by PaulStanos** [comment]
Why was this closed? Was it fixed?

**Comment by khussey** [comment]
Apologies, a little quick on the trigger finger. A fix for this issue will be included in a 1.5.2 release later this week.

**Comment by PaulStanos** [comment]
Not a problem. Glad to hear it's being worked on! Thanks for the update.

**Comment by khussey** [comment]
The fix is now available in the 1.5.2 release.

**Comment by Qasim-Aziz** [comment]
Client v1.14.2 (api v2)
Daemon v1.14.2 (api v2)

facing same problem , with edgectl login --namespace=ambassador example.com not found 404 over browser
